InternetShop/src/main/java/dao/OrderDaoFileImpl.java

```diff
@@ -0,0 +1,78 @@
+package dao;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import model.Order;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class OrderDaoFileImpl implements OrderDao {
+    private ObjectMapper mapper = new ObjectMapper();
+    private File file = new File("InternetShop/src/main/java/resource/orders.json");
+    private Map<Long, Order> ordersMap = new LinkedHashMap<>();
+
+    @Override
+    public List<Order> getAllOrders() {
+        readWithFile();
```

Comment:
plz distinct logic of initiating in-mem collection from file  and accessing in-mem collection itself.
I mean, when your DB will store 100,000 records, it will be very inefficient to read file on each operation